src/index.js

```diff
@@ -121,7 +121,7 @@ const mapToPropertyDescriptor = (
 
       const attributeValue = (() => {
         if (typeOrSerDes === Types.json) {
-          return JSON.stringify(value);
+          return typeof value === 'string' ? value : JSON.stringify(value);
```

Comment:
is `typeof value === 'string'` always true? How do you tell a non-JSON string from any other string?